src/typeahead/typeahead.ts

```diff
@@ -67,7 +67,7 @@ let nextWindowId = 0;
     '[class.open]': 'isPopupOpen()',
     '(document:click)': 'onDocumentClick($event)',
     '(keydown)': 'handleKeyDown($event)',
-    'autocomplete': 'off',
+    '[attr.autocomplete]': 'autocomplete',
```

Comment:
nit: there seems to be the `autcomplete` _property_ as well (https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement), so you could probably write it as `'[autocomplete]': 'autocomplete'`